packages/jsxstyle-utils/src/getStyleCache.ts

```diff
@@ -14,17 +34,17 @@ function alreadyInjected() {
   );
 }
 
-function getStringHash(key) {
+function getStringHash(key: string, _: {}) {
```

Comment:
Is there an official way to ignore a second parameter? It’s passed to `getClassNameForKey` on line 72 and I get why TS would complain about function signature being different, but an unused underscore param seems hella nasty. It gets minified out so I’m not too stressed about it, but just seems like there’s got to be a better way to handle this.

Reply:
Yeah prefix name with underscore and it will be ignored if unused

Reply:
You can use function overloads for this:
```ts
function getStringHash(key: string, props: any);
function getStringHash(key: string) {
  ...
}
```